### DISCUSSION TITLE
Changes of renal clearance rate in elderly 

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#841] [Questions & Problems] [answered]
hello, erveryone:
I have a question when I used pk-sim to simulate healthy elderly. healthy people's GFR was 120ml/min, and in "ADME" section , we all know , GFR decreases with age in the elderly, so what happens to the kidney clearance rate when I simulate the elderly?There was any specific relationship between  GFR and renal clearance rates when age changed?With age-related declines in GFR, there's any proportional decline in renal clearance?Or there is any age-related data on kidney clearance rates?

**Comment by prvmalik** [comment, marked as answer]
Age-related changes in GFR are accounted for by default in PK-Sim according to the work by Jan Schlender and team
doi: 10.1007/s40262-016-0422-3

**Comment by Wenjing1018** [comment]
@prvmalik ，
Your reply is very helpful for me,I wonder  kidney  plasma clearance rate  how to change under the premise of age-related changes in GFR  accounted for by default in PK-Sim？
Yours,
Sincerely!
